_posts/2017-12-08-totw-123.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Tip of the Week #123: `absl::optional` and `std::unique_ptr`"
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/123
+type: markdown
+---
+
+Originally posted as totw/123 on 2016-09-06
+
+By Alexey Skolov and Etienne Dechamps
```

Comment:
Add email addresses?

Reply:
Done (I was waiting for their permission).